cmake/RadiumSetupFunctions.cmake

```diff
@@ -185,29 +185,25 @@ function(configure_bundled_radium_app)
         endif()
     endforeach()
     list(REMOVE_DUPLICATES depsRsc)
-    # install qtPlugins (QPA plugin name found here https://doc.qt.io/qt-5/qpa.html) needed to fixup
-    # the bundle
+
+    # prepare to deploy using either macdeployQt or fixup-bundle
     get_target_property(IsUsingQt ${ARGS_NAME} LINK_LIBRARIES)
-    list(FIND IsUsingQt "Qt::Gui" QTGUIIDX)
-    if(NOT QTGUIIDX EQUAL -1)
-        message(STATUS "[configure_bundled_radium_app] installing Qt plugins for ${ARGS_NAME}.")
-        # Find the list of plugins needed for a macos Qt app and add them here
-        install_qt_plugin(
-            "Qt::QCocoaIntegrationPlugin" INSTALLED_QT_PLUGINS
-            "${CMAKE_INSTALL_PREFIX}/bin/${ARGS_NAME}.app/Contents/"
-        )
-        install_qt_plugin(
-            "Qt::QMacStylePlugin" INSTALLED_QT_PLUGINS
-            "${CMAKE_INSTALL_PREFIX}/bin/${ARGS_NAME}.app/Contents/"
-        )
+    list(FIND IsUsingQt "Qt::Core" QTCoreIdx)
+    list(FIND IsUsingRadiumGui "Radium::Gui" RadiumGuiIdx)
+    if(NOT ((QTCoreIdx EQUAL -1) AND (RadiumGuiIdx EQUAL -1)))
+        set(DeployWithQt TRUE)
+        # Retrieve the absolute path to qmake and then use that path to find the macdeployqt binary
+        get_target_property(_qmake_executable Qt::qmake IMPORTED_LOCATION)
+        get_filename_component(_qt_bin_dir "${_qmake_executable}" DIRECTORY)
+        find_program(MACDEPLOYQT_EXECUTABLE macdeployqt HINTS "${_qt_bin_dir}")
```

Comment:
This code is parsed **only** on macos.
It is located in the function `configure_bundled_radium_app` (define from line 128 to 264) and this function is only called at line 683 which is enclosed in a `if(APPLE)` (line 680) ... `elseif(MSVC ... )` (line 687).
So the answer is yes, this code is used only on macos.
BTW, there is room to improve the `RadiumSetupFunctions.cmake` so that it includes the system-dependant code (some functions are only for windows, other only for Linux, others for macos) only when needed and, when Radium is installed, generates a system-specific `RadiumSetupFunctions.cmake` instead of deploying all system independent logic. But this should be another PR.